### DISCUSSION TITLE
Can I opt to not use a Theme?

### DISCUSSION THREAD

**Discussion by RappyPhan** [Kaliumhexacyanoferrat/GenHTTP#331] [Q&A] [answered]
Hi there!

I started playing with GenHTTP to see if it would fit my needs. What I'm looking for is a web server that can use a light templating solution that leaves everything else to my own C# code to replace an aging ASP.NET front end.

There are several template engines available, and I chose Razor. But GenHTTP forced me to choose a Theme, which is basically wrapper HTML output by Scriban. Okay, let's make a bare-bones Theme to keep GenHTTP happy. But it insists on me providing styles and scripts even though the Theme doesn't use them.

It looks like what I want in the end is to not use Themes, or at the most a stripped-down version. Is this possible, or should I implement a custom handler to get more control?

I'd prefer not to implement a custom handler, as I fear it will lead to reinventing part of the wheel. :)

**Comment by Kaliumhexacyanoferrat** [comment, marked as answer]
Thanks for your question - I'm on vacation until Tuesday and will leave an answer next week 😉

**Reply by RappyPhan** [reply]
Are you still on vacation? :)

**Reply by RappyPhan** [reply]
Until which Tuesday are you on vacation?

**Reply by Kaliumhexacyanoferrat** [reply]
Excuse me, but the family needs a lot of time - find a new "NoTheme" here: https://www.nuget.org/packages/GenHTTP.Themes.NoTheme/
